client/src/shared/header/header.html

```diff
@@ -1,7 +1,7 @@
 <header class="wk-header" role="banner">
     <swt-hamburger-menu></swt-hamburger-menu>
     <div class="wk-header-container">
-        <a class="wk-logo" href="javascript:void(0)">
+        <a class="wk-logo" ui-sref="{{header.state}}">
```

Comment:
You can simply pass state name to ui-sref:
```html
<a class="wk-logo" ui-sref="home">
```
Please use this simplified version

Reply:
Updated!